firestore-bundle-server/extension.yaml

```diff
@@ -0,0 +1,120 @@
+# Learn detailed information about the fields of an extension.yaml file in the docs
+
+name: firestore-bundle-server
+specVersion: v1beta
+version: 0.1.0
+license: Apache-2.0
+
+displayName: Firestore Bundle Server
+
+description: >-
+  Provides cached serving of Firestore bundles.
+
+billingRequired: true
+
+# For your extension to interact with other Google APIs (like Firestore, Cloud Storage, or Cloud Translation),
+# set the `apis` field. In addition, set the `roles` field to grant appropriate IAM access to interact with these products.
+# Learn about these fields in the docs
+
+roles:
+- role: datastore.user
+  reason:
+    Allows the extension to read configuration and build bundles from Firestore.
+- role: storage.objectAdmin
+  reason:
+    Allows the extension to save built bundles in Cloud Storage
+
+# Learn about the `resources` field in the docs
+resources:
+  - name: serve
+    type: firebaseextensions.v1beta.function
+    description: >-
+      HTTPS function that serves bundled content from Cloud Storage cache or by dynamically building.
+    properties:
+      location: ${LOCATION}
+      httpsTrigger: {}
+      runtime: "nodejs12"
+
+# Learn about the `params` field in the docs
+params:
+  - param: LOCATION
+    label: Cloud Functions location
+    description: >-
+      Where do you want to deploy the functions created for this extension?
+      You usually want a location close to your database or Storage bucket.
+      For help selecting a location, refer to the [location selection
+      guide](https://firebase.google.com/docs/functions/locations).
+    type: select
+    options:
+    - label: Iowa (us-central1)
```

Comment:
It's my understand that functions only work on `us-central-1` servers, is still the case?

If so, would we still need a full list of locations.

![image](https://user-images.githubusercontent.com/2060661/112619055-bf06f780-8e1e-11eb-9e9b-292097176ea0.png)


Source: https://firebase.google.com/docs/hosting/full-config#rewrite-functions

Reply:
This is a good catch. Done.

Reply:
If `us-central1` is the only option, this can be removed as a param and set directly in `resources.properties.location`: https://github.com/FirebaseExtended/experimental-extensions/blob/f867cc07dd7d44330616b9ef6a9457cfde826d4a/firestore-bundle-server/extension.yaml#L34

Reply:
Done.